rudder-server-relay/SPECS/rudder-server-relay.spec

```diff
@@ -274,10 +274,13 @@ rm -rf %{buildroot}
 %config(noreplace) /etc/%{apache_vhost_dir}/rudder.conf
 %config(noreplace) /opt/rudder/etc/rudder-apache-relay-common.conf
 %config(noreplace) /opt/rudder/etc/rudder-apache-relay-ssl.conf
+%config(noreplace) /opt/rudder/etc/rudder-apache-relay-nossl.conf
 %config(noreplace) /opt/rudder/etc/rudder-networks-24.conf
 %config(noreplace) /opt/rudder/etc/rudder-networks-policy-server-24.conf
 %config(noreplace) /opt/rudder/etc/rudder-pkg/rudder-pkg.conf
+%config(noreplace) /opt/rudder/etc/rudder-pkg/rudder_plugins_key.pub
```

Comment:
this one should not be in noreplace, but it's the same for apache config and openssl config at least...